modules/create-wif-cluster-ocm.adoc

```diff
@@ -0,0 +1,226 @@
+// Module included in the following assemblies:
+//
+// * osd_install_access_delete_cluster/creating-a-gcp-cluster-with-workload-identity-federation.adoc
+
+
+:_mod-docs-content-type: PROCEDURE
+[id="create-wif-cluster-ocm_{context}"]
+= Creating a Workload Identity Federation cluster using {cluster-manager}
+
+.Procedure
+
+. Log in to {cluster-manager-url} and click *Create cluster* on the {product-title} tile.
+
+. Under *Billing model*, configure the subscription type and infrastructure type.
++
+[IMPORTANT]
+====
+Workload Identity Federation is supported by Customer Cloud Subscription (CCS) infrastructure type only.
+====
++
+.. Select a subscription type. For information about {product-title} subscription options, see link:https://access.redhat.com/documentation/en-us/openshift_cluster_manager/1-latest/html-single/managing_clusters/index#assembly-cluster-subscriptions[Cluster subscriptions and registration] in the {cluster-manager} documentation.
++
+
+.. Select the *Customer cloud subscription* infrastructure type.
+
+.. Click *Next*.
+. Select *Run on Google Cloud Platform*.
+. Select *Workload Identity Federation* as the Authentication type.
+.. Read and complete all the required prerequisites.
+
+.. Click the checkbox indicating that you have read and completed all the required prerequisites.
++
+. To create a new WIF configuration, open a terminal window and run the following OCM CLI command.
++
+
+[source,terminal]
+----
+$ ocm gcp create wif-config --name <wif_name> \ <1>
+  --project <gcp_project_id> \ <2>
+  --tag-enforcement=<true|false> \ <3>
+----
+<1> Replace `<wif_name>` with the name of your WIF configuration.
+<2> Replace `<gcp_project_id>` with the ID of the {GCP} project where the WIF configuration will be implemented.
+<3> Replace`<true|false>` with `false`.
++
```

Comment:
Should the following be included so its the same as for the CLI?

<4> Optional: Replace <required_versions> with the list of Openshift versions that the WIF configuration  needs to support. If no value is provided, the latest available version is supported by default.